src/sdk/sql_cluster_router.cc

```diff
@@ -1377,6 +1377,18 @@ base::Status SQLClusterRouter::HandleSQLCreateTable(hybridse::node::CreatePlanNo
     }
     ::openmldb::nameserver::TableInfo table_info;
     table_info.set_db(db_name);
+
+    if (!cluster_sdk_->IsClusterMode()) {
+        if (create_node->GetReplicaNum() != 1) {
+            return base::Status(base::ReturnCode::kSQLCmdRunError, 
+            "ERROR: Fail to create table with the replica configuration in standalone DB");
```

Comment:
indent.
u can format changes with clang-format

Reply:
> indent. u can format changes with clang-format. Also fix those cpplint warnings

ok I will update the format